lib/exercises.rb

```diff
@@ -2,19 +2,51 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n^2)
+# Space Complexity: O(n)
 
 def grouped_anagrams(strings)
```

Comment:
This works but it's O(n * m log m) where m is the length of the words or if the words are guaranteed to be sorter than some values O(n).